keylime/cloud_verifier_common.py

```diff
@@ -63,6 +63,8 @@ class CloudAgent_Operational_State:
     TERMINATED = 8
     INVALID_QUOTE = 9
     TENANT_FAILED = 10
+    GET_PROVIDER_QUOTE = 11
+    GET_PROVIDER_QUOTE_RETRY = 12
```

Comment:
Note that this will require some WebApp-related updates (though the existing one is being replaced eventually).  

Namely: 
https://github.com/keylime/keylime/blob/5f92021e64e6a08d5cf2f14b1f2e60293ddf60b8/keylime/static/js/webapp.js#L258-L283

and 

https://github.com/keylime/keylime/blob/5f92021e64e6a08d5cf2f14b1f2e60293ddf60b8/keylime/tenant_webapp.py#L402